.ebextensions/deploy.config

```diff
@@ -0,0 +1,118 @@
+option_settings:
+  - namespace: aws:autoscaling:launchconfiguration
+    option_name: BlockDeviceMappings
+    value: /dev/sdh=:10
+  - option_name: BUNDLE_DISABLE_SHARED_GEMS
+    value: "1"
+  - option_name: BUNDLE_PATH
+    value: "vendor/bundle"
+
+packages:
+  yum:
+    git: []
+
+files:
+  "/etc/nginx/conf.d/proxy.conf" :
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      client_max_body_size  200M;
+      proxy_connect_timeout 1800;
+      proxy_send_timeout    1800;
+      proxy_read_timeout    1800;
+      send_timeout          1800;
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/03_webapp_user.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      #!/usr/bin/env bash
+      sudo chown -R webapp:webapp /home/webapp
+      sudo service nginx reload
+      true
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/13_ebs_perm.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      #!/usr/bin/env bash
+      if ! mount | grep /dev/xvdh > /dev/null; then
```

Comment:
We're making several assumptions here, and I don't know how valid they are. We're assuming that the device we're looking for is /dev/xvdh and it's mounted on /media/volume1. This might be a reliable assumption to make, but it might not.